tasks/release.py

```diff
@@ -665,6 +665,13 @@ def _get_windows_ddnpm_release_json_info(
     return win_ddnpm_driver, win_ddnpm_version, win_ddnpm_shasum
 
 
+@task
+def get_variable(_, name, version='nightly'):
+    with open("release.json", "r") as release_json_stream:
+        release_json = json.load(release_json_stream, object_pairs_hook=OrderedDict)
+    print(release_json[version][name])
+
+
```

Comment:
I don't think this is used anytwhere, no ?

Reply:
It is used to set several env variables in gitlab pipelines. For example: https://github.com/DataDog/datadog-agent/pull/8585/files#diff-45842dbfad3838a05ba2c9487beb8f2e2ef0b6d21cd586dc60d5224cd3626fe2R106